ethabi/src/encoder.rs

```diff
@@ -37,26 +37,38 @@ enum Mediate {
 	Prefixed(Vec<[u8; 32]>),
 	FixedArray(Vec<Mediate>),
 	Array(Vec<Mediate>),
+    RawTuple(Vec<Mediate>),
```

Comment:
Could we get a comment somewhere on what the difference between raw and prefixed is?